sdk/python/tests/test.py

```diff
@@ -107,6 +107,27 @@ def test_log_invalid_artifacts_should_fail(self):
     ))
     self.assertRaises(ValueError, e.log_output, artifact2)
 
+  def test_log_metadata_successfully_with_minimum_inforamtion(self):
+    ws1 = metadata.Workspace(backend_url_prefix="127.0.0.1:8080", name="ws_1")
+
+    r = metadata.Run(workspace=ws1, name="first run")
+
+    e = metadata.Execution(name="test execution", workspace=ws1, run=r)
+    assert e.id
```

Comment:
Are we checking that this is not none or true? If so, prefer `self.assertIsNotNone(e.id)` or similar.

Same below

Reply:
Done!